app/src/main/java/ch/nevis/exampleapp/coroutines/domain/util/Authenticator.kt

```diff
@@ -0,0 +1,27 @@
+package ch.nevis.exampleapp.coroutines.domain.util
+
+import ch.nevis.mobile.sdk.api.localdata.Authenticator
+import ch.nevis.mobile.sdk.api.localdata.UserEnrollment
+
+/**
+ * Extension function of [Authenticator] to check if it's enrolled for the given user/account or not.
+ *
+ * @param username The username that identifies the user/account.
+ * @param allowClass2Sensors A flag that tells whether using Class 2 biometric sensors is allowed
+ * if the biometric authenticator is selected.
+ * @return A flag that tells whether the authenticator is enrolled for the user or not.
+ * @throws IllegalStateException if the [Authenticator] has an unknown [UserEnrollment] implementation.
+ */
+fun Authenticator.isUserEnrolled(username: String, allowClass2Sensors: Boolean): Boolean {
```

Comment:
This is duplicated, exactly the same extension can be found in the `ch.nevis.exampleapp.domain.util` package. Do we need both of them?

Reply:
Thanks for spotting this. I've deleted the one from `ch.nevis.exampleapp.data.util`. It was accidentally copied to there.